packages/manager/modules/nutanix/src/dashboard/general-info/redeploy/confirm/template.html

```diff
@@ -0,0 +1,35 @@
+<form
+    id="nutanixRedeployConfirm"
+    name="nutanixRedeployConfirm"
+    data-ng-submit="nutanixRedeployConfirm.$valid && $ctrl.redeployCluster()"
+    novalidate
+>
+    <oui-modal
```

Comment:
Why not using directly `data-primary-action` here? This way you can disable also the validate button if pattern doesn't match rule. Here you allow the user to click on button is `redeploy` is filed, but cancel the action if it is not valid